.pre-commit-config.yaml

```diff
@@ -3,25 +3,25 @@
 
 repos:
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.15.2
+    rev: v3.19.1
     hooks:
       - id: pyupgrade
         args:
-          - --py38-plus
-  - repo: https://github.com/pycqa/flake8
-    rev: 7.0.0
-    hooks:
-      - id: flake8
-        language_version: python3
+          - --py39-plus
```

Comment:
As mentioned above, I think we should stick with 3.9, but if we do enforce a 3.10 minimum, we should set that here too